crates/ra_ide/src/display/navigation_target.rs

```diff
@@ -150,6 +151,26 @@ impl NavigationTarget {
         )
     }
 
+    /// Allows `NavigationTarget` to be created from a `DocCommentsOwner` and a `NameOwner`
+    pub(crate) fn from_doc_commented(
+        db: &RootDatabase,
+        named: InFile<&dyn ast::NameOwner>,
+        node: InFile<&dyn ast::DocCommentsOwner>,
+    ) -> NavigationTarget {
+        let name =
+            named.value.name().map(|it| it.text().clone()).unwrap_or_else(|| SmolStr::new("_"));
+        let focus_range = node.value.doc_comments().next().map(|it| it.syntax().text_range());
+        let frange = original_range(db, node.map(|it| it.syntax()));
+
+        NavigationTarget::from_syntax(
+            frange.file_id,
+            name,
+            focus_range,
+            frange.range,
```

Comment:
Why does this set both `full_range` and `focus_range`? What is the semantics of focus range for a piece of doc comments? Seems like we should leave `focus_range`  as `None`

Reply:
fixed